Sophmore page

jay-deep opened the pull request:
Bro , Now check it, is it fine?
Will definitely enhance the css but main thing is code, I tried so hard , You assigned me so literally I dont wanted to let u down, Plz inform me the errors in this.
Fixes #3

Commit:
Sophmore page

Bro , Now check it, is it fine?
Will definitely enhance the css but main thing is code, I tried so hard , You assigned me so literally I dont wanted to let u down, Plz inform me the errors in this.
#20

jay-deep commented:
Bro can u add hactobefest in your label topic?

decon-harsh commented:
See @jay-deep This PR is quite good, Here are few things i would suggest..

- Copy the navbar code from index.html 
- Link stylesheet styles.css to sophomore pages (don't write same css multiple times)
- Copy whole head from index.html
- Write custom css for some parts, tryna use bootstrap defined classes.
- Avoid this much use of br tags.
- Dont add css directly in html file , write css in a file sophomore.css and place it in css directory , link the stylesheet from there. 


- Dont open a new pr make changes in your local , push it your fork it will automatically reflect in this PR

decon-harsh commented:
I added these tags for now but you have to complete this, or else I am forced to close this. Thank you :)

jay-deep commented:
Bro actually , right now I don't know how to use bootstrap
I have did some changes that u suggested

Linked new sophomore.css

added navbar code from index.html

Removed some br

and bro I pushed it to my fork

Commit:
Enhancement

decon-harsh commented:
That's great, so do you want to enhance it's CSS or should I take it from here. Work both ways
Since you had completed hacktober, I would encourage you to take this up as a new thing to learn. It will increase your skills surely.

jay-deep commented:
yeah definitely bro, it was never about hacktober fest, its just as like till 1st october , I didnt even knew how to use github, now atleast I am happy that I know a little bit. Bro Do 1 thing na, You Merge this in master and if I got something to enhance in it, I will do it parallely? What say?

decon-harsh commented:
> yeah definitely bro, it was never about hacktober fest, its just as like till 1st october , I didnt even knew how to use github, now atleast I am happy that I know a little bit. Bro Do 1 thing na, You Merge this in master and if I got something to enhance in it, I will do it parallely? What say?

This will mess the commit logs , multiple commits for same work is not appreciated. However if you want I can merge this is new branch and when done , will merge this in master will a single commit(Squashing) .

jay-deep commented:
Yup that's also great

decon-harsh commented:
I checked once merged , i guess you can't edit the commits . Then you have to send multiple PRs that's not cool. 

If a PR is not merged and you add new commit it will automatically show.  Better to not merge it now.

jay-deep commented:
But now, how will u add this sophomore page in master branch?

decon-harsh commented:
> But now, how will u add this sophomore page in master branch?

I will wait till you complete this task ;)

jay-deep commented:
Now I just need to enhance it na, I mean no need to add something na???

decon-harsh commented:
> Now I just need to enhance it na, I mean no need to add something na???

Yes enhance the styles

Commit:
Css enhancement

Did some enhancement in css

jay-deep commented:
Bro I did some enhancements , I don't think more enhancement is needed in this, Yes if there's something that u want to add then plz inform and bro when u merge it na plz put hacktoberfest-accepted label on it
![Capture](https://user-images.githubusercontent.com/44156426/95240371-edb9f300-0829-11eb-8559-96b5cb01668b.PNG)
Because I think according to new rules only the PR with these label are counted

decon-harsh commented:
![Screenshot_2020-10-07-00-55-54-648_com.github.android.jpg](https://user-images.githubusercontent.com/58214248/95250545-e1d52d80-0837-11eb-9314-083023911622.jpg)

The labels are already there

jay-deep commented:
okay cool, bro Did u checked the enhancements?
